vtex-apps/shipping-calculator
# CHK-440: Feat/pickup options

#### What problem is this solving?

This PR is part of the work needed in order to enable the selection of a pickup point as a shipping option.



#### How to test it?

It depends on and should be tested alongside the following PRs:

https://github.com/vtex-apps/checkout-graphql/pull/107
https://github.com/vtex-apps/checkout-shipping/pull/18
https://github.com/vtex-apps/checkout-resources/pull/61
https://github.com/vtex-apps/order-shipping/pull/14
